src/video/sdl2_v.h

```diff
@@ -57,6 +58,10 @@ class VideoDriver_SDL_Base : public VideoDriver {
 	void CheckPaletteAnim() override;
 	bool PollEvent() override;
 
+	bool OpenGamepad() override;
+	void CloseGamepad() override;
+	void ProcessGamepadInput() override;
+
```

Comment:
Declare the helper function, -1 means an invalid id, I couldn't find a constant for that in SDL.
```suggestion
	bool OpenGamepadByJoystickID(SDL_JoystickID joyid = -1);

```